system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -75,4 +77,43 @@
     margin: 0;
   }
 
+  .section-empty-state {
+    width: 100%;
+    height: 266px;
```

Comment:
Can you refactor `$card-height` to be available to both this file and _Card.scss so you can use it here?